src/views/Character.vue

```diff
@@ -614,11 +614,11 @@ export default {
       streamOverlay: state => state.app.windowStyle === 'overlay',
       maxExperience: state => levelExperience[state.ws.character.level],
       experiencePercentage(state) {
-        const prev = levelExperience[state.ws.character.level - 1];
-        const next = levelExperience[state.ws.character.level];
+        const { level, experience } = state.ws.character;
+        const prev = levelExperience[level - 1];
+        const next = levelExperience[level];
+        const p = ((Number(experience) - prev) / (next - prev)) * 100;
```

Comment:
could you explain (or link an explanation) why `Number(experience)` fixes the problem?

Reply:
The problem comes from npm's postgresql package `pg` returning `bigint` values from the database as strings to prevent precision loss. The diablorun database does not really need absolute precision for any `bigint` values that are used so if possible then this fix could be better on the server side instead. It should be possible to configure `pg` to return `bigint` values as doubles instead.

Reply:
Interesting, thanks :+1: